ggp-engine/DebugScene.cpp

```diff
@@ -144,33 +166,13 @@ void DebugScene::Update(float _deltaTime) {
 
 	GetGameObject<Spatial>("Object5")->transform.position.x = 2.0f * sin(totalTime);
 
-	GetGameObject<Spatial>("light2")->transform.position.z = 2.0f * sin(totalTime);
-
-    // For every physics object in the scene
-
-    /*
-    if ( entityA->GetPhysicsLayer() == EPhysicsLayer::STATIC ) continue;
+	GetGameObject<Spatial>("light2")->transform.position.z = 2.0f * sin(totalTime);*/
 
-    XMFLOAT3 forceToApply = XMFLOAT3( 0.f, 0.f, 0.f );
-    forceToApply.y += Gravity;
 
-    forceToApply.x *= deltaTime;
-    forceToApply.y *= deltaTime;
-    forceToApply.z *= deltaTime;
 
-    entityA->ApplyForce( forceToApply );
-
-    if ( entityA->GetPosition().y < -2.f )
-    {
-        XMFLOAT3 opposingVel = entityA->GetVelocity();
-        opposingVel.x *= -1.f;
-        opposingVel.y *= -1.f;
-        opposingVel.z *= -1.f;
-        entityA->ApplyForce( opposingVel );
-    }
-
-
-
-    entityA->ApplyAcceleration();*/
+    // For every physics object in the scene
+    // Update the physics
+    physicsManager->UpdatePhysics( _deltaTime );
```

Comment:
This should probably be moved to the Game.cpp with the rest of the singleton update calls. We want scenes to be decoupled from singletons.